### ISSUE TITLE
internal function now() returns unexpected result

### ISSUE THREAD

**Issue by rebrendov** [ClickHouse/clickhouse-go#252]
Hello!
In [this](https://github.com/ClickHouse/clickhouse-go/blob/master/bootstrap.go#L51) line `unixtime` is incremented by int64(time.Second) which is evaluated to 1000000000
https://play.golang.org/p/OgfYtAUqV6w
And then in function `now()` there is a `time.Unix()` where `unixtime` is passed as seconds https://github.com/ClickHouse/clickhouse-go/blob/master/bootstrap.go#L58
That leads to `now()` returning rapidly increasing results.
TL;DR unixtime is incremented by 1000000000 per second.
